src/mcp/server/fastmcp/utilities/func_metadata.py

```diff
@@ -80,7 +80,7 @@ def pre_parse_json(self, data: dict[str, Any]) -> dict[str, Any]:
         dicts (JSON objects) as JSON strings, which can be pre-parsed here.
         """
         new_data = data.copy()  # Shallow copy
-        for field_name, field_info in self.arg_model.model_fields.items():
+        for field_name, _field_info in self.arg_model.model_fields.items():
```

Comment:
sounds like we can make this
```suggestion
        for field_name in self.arg_model.model_fields.keys():
```